web/html/src/components/panels/InnerPanel.tsx

```diff
@@ -5,26 +5,41 @@ import { SectionToolbar } from "components/section-toolbar/section-toolbar";
 type Props = {
   title: string;
   icon: string;
-  buttons: React.ReactNode;
-  buttonsLeft?: React.ReactNode;
+  buttons: React.ReactNode[];
```

Comment:
Is there a reason to only limit this to arrays? If we want to check for non-empty arrays we could do something along the lines of

```tsx
if (typeof buttons === 'undefined' || (Array.isArray(buttons) && buttons.length === 0)) {
  return null;
}
return ...;
```

Reply:
I kind of wanted to limit it because the props is `buttons` so I expect a collection (`Array`) of items. And we also render them already into a `<div className="btn-group">` button wrapper. So I want to make sure we shape the prop in the right way otherwise we do not know what to expect and what will be rendered in the end.

Reply:
I more so meant that code-wise there will be no difference, both will still be rendered using `{props.buttons}`, so I was wondering if there was something I was missing. I don't have a strong preference though, this works well too.